src/basic_recipes/contourf.jl

```diff
@@ -141,12 +135,14 @@ function AbstractPlotting.plot!(c::Contourf{<:Tuple{<:AbstractVector{<:Real}, <:
 
     poly!(c,
         polys,
-        # colormap = c._computed_colormap,
-        # colorrange = colorrange,
+        colormap = c._computed_colormap,
+        colorrange = colorrange,
```

Comment:
I want to highlight this because I've done more than just add `inspectable` here. I don't really use contours so maybe someone else can check that these changes are fine.